.ci/common.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# Attention, there is no "-x" to avoid problems on Travis
+set -e
+
+DEBUG=true
+
+# result of command goes to $RUN_JOB
+# 0 = not run (skip)
+# 1 = run (no skip)
+function should_run_job {
+    local SKIP_JOB_BY_FILES=$1
+    local SKIP_FILES=$2
+    local OUT=""
+
+    echo "Initial value of SKIP_JOB_BY_FILES: $SKIP_JOB_BY_FILES"
+    echo "Files that are allowed to be skipped: $SKIP_FILES"
+
+    if [[ $SKIP_JOB_BY_FILES != 'false' ]]; then
+         ## Identify previous commit to know how much to check
+         ## Check assumes we are only working with 1 commit if we are in master
+         local PREVIOUS_COMMIT="HEAD~1"
+
+         if [[ $DEBUG == "true" ]]; then
+              OUT=`git rev-parse HEAD`
+              echo "Head commit: $OUT"
+         fi
+         if [[ $DEBUG == "true" ]]; then
```

Comment:
Two debug blocks. Please fix.

I still think all will benefit if always print this information to console, without any condition.

Reply:
I merged them into 1, but it kills wercker.
I don't understand why but the other CIs continue to work fine.